string[] classifier_names
uint32[] predictions
float32[] p_votes
uint32[] sizes
float32[] t_votes
